packages/orchestration/src/utils/orchestrationAccount.js

```diff
@@ -13,6 +13,17 @@ import {
 
 const { Vow$ } = NetworkShape; // TODO #9611
 
+/**
+ * Mark a possibly incomplete object as complete by removing the `incomplete`
+ * property.
+ *
+ * @template {Record<string, any>} T
+ * @param {T & { incomplete?: any }} maybeIncomplete
+ * @returns {T}
+ */
+export const complete = ({ incomplete: _, ...completed }) =>
```

Comment:
this verb sounds like it's doing something with the object. The past participle sounds like you're getting back what was done:

consider,
```suggestion
export const completed = ({ incomplete: _, ...completed }) =>
```